src/go/configmanager/testdata/test_fixed_mode_dynamic_routing.go

```diff
@@ -599,6 +608,7 @@ var (
                         ],
                         "path": "/search"
                       },
+											"name": "1.echo_api_endpoints_cloudesf_testing_cloud_goog.dynamic_routing_Search",
```

Comment:
NIT, indent

Reply:
Done. I didn't notice this. My intellij's indent showed two spaces.